test/parser.test.ts

```diff
@@ -1059,6 +1079,28 @@ describe('Wollok parser', () => {
 
   })
 
+  describe('Fixture', () => {
```

Comment:
You totally got the idea of which scenarios we want to cover. This are the perfect test cases for this feature. Good Job.